app/assets/javascripts/discourse/models/post.js.es6

```diff
@@ -320,6 +320,13 @@ Post.reopenClass({
     });
   },
 
+  mergePosts(selectedPosts) {
+    return Discourse.ajax("/posts/merge_posts", {
+      type: 'PUT',
+      data: {post_ids: selectedPosts.map(p => p.get('id'))}
+    });
```

Comment:
Can you tell me what I should do when if there's an error? There's no error handling in any of the AJAX calls of that model.